### PR TITLE
fixes for noetic build

### PR DESCRIPTION
**Pull request by davefeilseifer**
these changes make pr2_ethercat_drivers work with noetic/python3. Cannot prerelease test, yet since ros-noetic-eml has not yet synced. However, it passes local tests.

### CODE REVIEW

File: fingertip_pressure/scripts/rectangle_viz.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python
```
Comment (v4hn) [inline fingertip_pressure/scripts/rectangle_viz.py]: https://github.com/PR2/linux_networking/pull/5#discussion_r584112906 for all of these.

File: fingertip_pressure/scripts/sim_sensor.py [hunk 2]
```diff
@@ -57,20 +57,20 @@ def callback(self, pressurestate):
     def publish(self):
         ps = PressureState()
         ps.header.stamp = rospy.get_rostime();
-        ps.l_finger_tip = []
-        ps.r_finger_tip = []
```
Comment (v4hn) [inline fingertip_pressure/scripts/sim_sensor.py]: I don't see why this change would do anything?

**Comment by v4hn** [comment]
Aside from my inline comments the actual changes all look good.
However, you mixed the Changelog stuff and multiple changes of the maintainer in there.
The changelog should only be edited during the release process, so could you please rebase this pr and remove all commits that do not change functionality?

File: fingertip_pressure/scripts/rectangle_viz.py (hunk 1 above)
Comment (davefeilseifer) [reply fingertip_pressure/scripts/rectangle_viz.py]: yeah, I figured this one out, it turns out I had python-is-python2 installed, which messes with the python stuff. Installing python3-is-python under ubuntu fixes this issue, and the shebang works as intended, now.

File: fingertip_pressure/scripts/sim_sensor.py (hunk 2 above)
Comment (davefeilseifer) [reply fingertip_pressure/scripts/sim_sensor.py]: same as issue above, ensuring python3 is used seems to fix the issue, this can revert

**Comment by k-okada** [comment]
@davefeilseifer thanks for fix, I have include your python3 fix into #79 , except using `catkin_instal_python` instaed of using `#/usr/bin/env python3`
